wordpress-mobile/WordPress-Android
# Issue/7463 post pubdate settings gone on rotate

**NOTE: #7541 should be merged before this is reviewed**

Along with #7541 this fixes #7463. This PR addresses the disappearing post settings date/time picker when the device is rotated.

To test:
- Open post settings
- Tap Publish
- Rotate the device

Notice that the dialog picker continues showing and retains the current selection
